mafiasi/gprot/models.py

```diff
@@ -6,17 +6,38 @@
 
 from mafiasi.teaching.models import Course, Teacher
 
+
 def make_filename(obj, prefix, ext):
     return '{0}/{1}-{2}.{3}'.format(
         prefix, hexlify(os.urandom(16)), obj.pk, ext)
 
+
 def make_gprot_filename(gprot, filename):
     return make_filename(gprot, 'gprot', 'pdf')
 
+
 def make_attachment_filename(attachment, filename):
     ext = attachment.mime_type.split('/')[-1]
     return make_filename(attachment, 'gprot-attachment', ext)
 
+
+class LabelManager(models.Manager):
+
+    def as_choices(self):
+        for label in self.all().order_by('tag'):
+            yield label.pk, label.tag
+
+
+class Label(models.Model):
+    tag = models.CharField(max_length=50)
```

Comment:
I would call this field `name`. Or is `tag` not just synonymous to `label` but instead a different semantics?
Also, cf. below regarding whether they are called label or tag – I would prefer using only one of the two terms consistently.